### ISSUE TITLE
Status indicators should be visually separated from actions in the "Servers Status" section of the dashboard

### ISSUE THREAD

**Issue by nekohayo** [webmin/webmin#2011]
A tiny "eyecandy" papercut issue, but now I can't un-see it :)  
In the Webmin 2.101 dashboard with the default Authentic theme, we have something like this:

![image](https://github.com/webmin/webmin/assets/479401/cb1f06c3-dbf7-4aff-94d3-31bc350e03c1)

...where the status of those services ("Servers Status") is indicated by a green checkmark  
...right next to the red "Stop" icon button and the blue "Restart" icon button.

When you stop one of the services, the UI becomes like this, it's mixing even more visually confusing things together in the same spot, and they're not really aligned:

![image](https://github.com/webmin/webmin/assets/479401/cb1e9a6b-d758-4dac-b7dd-3bebdeed9282)

There are a few UX issues with this:
* Unlike the other two, the green checkmark status icon has no tooltip on hover
* It is located in the same spot, so it gets visually confounded as an action rather than a status indicator; it might be better to locate it to the far left, before the service name?
* The use of icons here overall is a bit inconsistent with the rest of the UI/theme, which typically uses labelled buttons (we can even see an example of that in the Actions column of the section just above, "Recent logins".
* Likewise, the "Network Interfaces" section has an "Active" column that uses colored text instead of icons to indicate status; maybe a consideration for the "Servers Status" section as well? Though I would understand if you prefer to keep using the green checkmark icon for space savings instead.

Depending on what you want to consider among possible solutions, one of the potential approaches could be this combination:
* Green checkmark vs red stop sign (instead of nothing) as an icon in a column on the left of the service name
* Stop / Start / Restart icon buttons replaced by labelled buttons, and only the relevant ones shown (i.e.: no "Restart" or "Stop" when the service is not running, only "Start")

I hope you find this suggestion helpful :)

**Comment by jcameron** [comment]
Yeah, we could definitely space these out a bit more for starters..

I'll have to think about switching away from icons though.

**Comment by iliajie** [comment]
> I'll have to think about switching away from icons though.

Why switching from icons?



> Yeah, we could definitely space these out a bit more for starters.

When the service is stopped there still should be the last icon (in place of restart) but empty and transparent (just to occupy the space), to push elements on the left consistently.

**Comment by jcameron** [comment]
> When the service is stopped there still should be the last icon (in place of restart) but empty and transparent (just to occupy the space), to push elements on the left consistently.

I think the original issue was about the space between the status and action icons, which I've fixed up ...

**Comment by nekohayo** [comment]
> Why switching from icons?

I think it could be worth considering for a couple of reasons:
* For a clearer visual distinction between "status indication" vs "interactive action widgets", beyond spacing. Buttons with labels usually convey clickability better. I just tend to think that if _everything_ is icons there, it gets less clear what is and isn't interactive, especially when some of those icons look similar to each other (ex: `(x)` in a red circle vs `(-)` in a red circle, green checkmark vs green triangle, ...).
* For coherence, as many actions buttons throughout webmin use labelled buttons
* Generally speaking, labelled buttons tend to have better accessibility (and assistive technologies compatibility) than icons, I think?
* You're not really space-constrained there, there's a ton of empty horizontal space ;)
  * It also opens-up the future possibility for you (if you want it) to add some more actions that may not be easy to convey with icons, like "View logs" (if one would want to directly jump to a realtime view of the logs provided by systemd/journald for that service, maybe)

**Comment by iliajie** [comment]
Alright, I have added more fixes to align status icons despite if service is running or not, e.g.:

Before:
<img width="808" alt="image" src="https://github.com/webmin/webmin/assets/4426533/7f39e9e9-2d85-4b1c-bc53-2d8c1ec1aa1b">

After:
<img width="808" alt="image" src="https://github.com/webmin/webmin/assets/4426533/c689feef-7bdf-433b-aac1-a5293aceefb8">

**Comment by iliajie** [comment]
I've also fixed the colors for stop icons and made icons consistent (for stopped services) with System and Server Status module.

<img width="857" alt="image" src="https://github.com/webmin/webmin/assets/4426533/2b2cab8e-2010-4f53-b9b2-aa53324579cf">
